Project/index.mjs

```
@@ -78,6 +78,113 @@ export default class Project {
             userMessage(error.message)
         }
     }
+
+    async makeLeader(userId) {
+        try {
+            const token = TPEN.getAuthorization() ?? TPEN.login()
```

These are generally helpful, but the attachAuth() method should ensure that these always have the data we're looking for. I might be willing to let this error out if someone forgot to fire a login before attempting this method.